bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,121 @@
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+
+from discord import Color, Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+TOP_WORDS_FILE_PATH = Path("bot/resources/fun/top_1000_used_words.txt").resolve()
+# defining a list of images that will be used for the game to represent the hangman person
+IMAGES = [
```

Comment:
Maybe upload the images not on discords CDN, because deletion would cause problems.

Reply:
I uploaded the images myself on my test server so I doubt they would ever get deleted, but would there be an alternative to discord's CDN?

Reply:
Image-to-link sharing services like imgur, perhaps. The best way would probably be to store it locally, in a folder in the `resources` folder or something like that.